src/test/java/com/microsoft/azure/relay/SendReceiveTest.java

```diff
@@ -280,29 +339,36 @@ private static void httpRequestHandler(RelayedHttpListenerContext context, Strin
 	}
 	
 	private static void httpRequestSender(String method, String msgExpected, String msgToSend) throws IOException {
+		HttpURLConnection connection = getHttpConnection();
+		sendBytesOverHttp(connection, msgToSend.getBytes(), 0, msgToSend.length());
```

Comment:
>msgToSend.length() [](start = 57, length = 18)

call msgToSend.getBytes() (UTF-8?), hang on to the byte[] and use its length here instead of String.length() (the UTF-16 character count).